### ISSUE TITLE
This GitLens pre-release version ... has expired. Please upgrade to a more recent version

### ISSUE THREAD

**Issue by Tyriar** [gitkraken/vscode-gitlens#2246]
### Description

I installed the pre-release GitLens extension and was presented with this:

![image](https://user-images.githubusercontent.com/2193314/193869895-a4912daa-e3fc-4d34-9207-10d6960893c9.png)

It is the latest, I have no idea what it's meant to mean?

### GitLens Version

v2022.8.3105

### VS Code Version

Version: 1.72.0-insider (user setup)
Commit: bb7692da16e8d4e01aea254d2f28312a673f351c
Date: 2022-10-04T05:43:53.890Z
Electron: 19.0.17
Chromium: 102.0.5005.167
Node.js: 16.14.2
V8: 10.2.154.15-electron.0
OS: Windows_NT x64 10.0.22621
Sandboxed: Yes

**Comment by SuperSamus** [comment]
Probably duplicate of #2229.

**Comment by eamodio** [comment]
So that is an unfortunate side effect of me trying to use pre-releases and ultimately having to abandon them because of too many unforeseen issues (mainly because of the lack of marketplace support). And I've been waiting on the marketplace to remove those pre-release versions (authors can't and apparently there are extra complications there 😞).

You'll need to either use the main version or use the latest insiders (non-pre-release) version.

**Comment by Tyriar** [comment]
@eamodio couldn't you release a new pre-release with a better tailored message? To me it looks like the right choice is the pre-release one over insiders as it's the newer feature.

**Comment by eamodio** [comment]
I could, but then I'm just creating more releases that need to be removed by the marketplace. I'm hoping this will get resolved soon. Right now there are too many caveats with pre-releases for GitLens to fully adopt them (unless we end up being forced into it if the marketplace can't remove them)

**Comment by Tyriar** [comment]
Up to you, my experience with this sort of thing is it can take a while 🙂

**Comment by eamodio** [comment]
This has been resolved now, since GitLens now using pre-releases.
